modules/vaos/app/serializers/vaos/v2/eps_appointment.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module VAOS
+  module V2
+    class EpsAppointment
+      def initialize(params = {})
+        appointment_details = params[:appointment_details]
+        referral_details = params[:referral]
+
+        @id = params[:id]&.to_s
+        @status = appointment_details[:status],
+        @patient_icn = params[:patient_id]
+        @created = appointment_details[:last_retrieved]
+        @location_id = params[:network_id]
+        @clinic = params[:provider_service_id]
+        @start = params.dig(:appointment_details, :start)
+        @contact = params[:contact]
+        @referral_id = referral_details[:referral_number]
+        @referral = { referral_number: referral_details[:referral_number]&.to_s }
+      end
+
+      def serializable_hash
+        {
+          id: @id,
+          status: determine_status(@status),
+          patient_icn: @patient_icn,
+          created: @created,
+          location_id: @location_id,
+          clinic: @clinic,
+          start: @start,
+          contact: @contact,
+          referral_id: @referral_id,
+          referral: @referral
+        }.compact
+      end
+
+      private
+
+      def determine_status(status)
+        status || 'proposed'
```

Comment:
I don't think this is still quite right. `status` as a string will always be truthy, and will return status unless it's `nil`.

```
vets-api(dev)> 'asdf' || 'proposed'
=> "asdf"
vets-api(dev)> nil || 'proposed'
=> "proposed"
vets-api(dev)> 'not a valid status' || 'proposed'
=> "not a valid status"
```

If you want status to be 'proposed' unless it's specifically 'booked', this should work:
```suggestion
        status == 'booked' ? 'booked' : 'proposed'
```